src/main/java/com/programmers/exception/ErrorChecking.java

```diff
@@ -0,0 +1,26 @@
+package com.programmers.exception;
+
+public class ErrorChecking {
+
```

Comment:
```suggestion
    private ErrorChecking() {}
```

유틸 클래스를 사용하는 경우 private 생성자를 통해 불필요한 객체가 생성되지 않도록 하는 것이 좋습니다.!

Reply:
[[refactor] 3. 유틸 클래스 기본 생성자의 접근제어자 변경
](https://github.com/prgrms-be-devcourse/java-library-management/commit/9fdf074903999260c9e3348a9d805558b19e441d)

반영 완료했습니다!